src/status_im/test_runner.cljs

```diff
@@ -4,7 +4,11 @@
             [clojure.string :as string]
             [shadow.test :as st]
             [shadow.test.env :as env]
-            [utils.re-frame :as rf]))
+            [utils.re-frame :as rf]
+            [i18n.i18n :as i18n]
+            [status-im2.setup.i18n-resources :as i18n-resources]))
+
+(i18n/init (name i18n-resources/default-device-language) (clj->js i18n-resources/translations-by-locale))
```

Comment:
I had to add this here as this used to be called in status-im/i18n/i18n.cljs.. it was not getting called for tests so I included it here 👍